netlify.toml

```
@@ -14,7 +14,13 @@ ignore = "git diff --quiet $CACHED_COMMIT_REF $COMMIT_REF . ../netlify.toml"
 HUGO_VERSION = "0.111.3"
 HUGO_ENABLEGITINFO = "true"
 
-[context.deploy-preview.environment]
+# On netlify our branch will always be the one we are currently building for
+# important information regarding naming 
+# https://gohugo.io/getting-started/configuration/#configure-with-environment-variables
+HUGOxPARAMSxGITHUB_BRANCH="$BRANCH"
+HUGOxPARAMSxCURRENTVERSION="$BRANCH"
```

we don't need one line of those anymore

```suggestion
HUGOxPARAMSxCURRENTVERSION="$BRANCH"
```